### DISCUSSION TITLE
Inquiry regarding Changing Solver Options in Colab

### DISCUSSION THREAD

**Discussion by HyemiK1m** [coin-or/Couenne#73] [General]
I tried to change the option, such as feas_tolerance, of couenne solver (global solver of MINLP). I used Pyomo and Colab. Does anyone know how to change it?

I tried this step:

solver = SolverFactory(solver)
solver.options['feas_tolerace'] = 1 # any value...
results = solver.solve(model, timelimit=10)
However, I got this log: 

ERROR:pyomo.opt:Solver (asl) returned non-zero return code (1) ERROR:pyomo.opt:Solver log: Couenne 0.5.8 -- an Open-Source solver for Mixed Integer Nonlinear Optimization Mailing list: [couenne@list.coin-or.org](mailto:couenne@list.coin-or.org) Instructions: http://www.coin-or.org/Couenne couenne: Unknown keyword "feas_tolerance" Unknown keyword "feas_tolerance"

I used feas_tolerance because it is specified in the user manual.

How can I fix this?

Thank you for reading my question :)

**Comment by merraksh** [comment]
Hi,

indeed feas_tolerance is a legitimate Couenne option and Couenne uses it when working standalone as it reads the couenne.opt file. This might be a Pyomo problem, i.e. maybe Pyomo is not currently configured to read an set one or more Couenne options. Perhaps it would be better to ask the Pyomo developers.

Hope this helps.

**Reply by HyemiK1m** [reply]
thank you!
